cockatrice/src/tab_supervisor.cpp

```diff
@@ -509,17 +509,30 @@ void TabSupervisor::processUserMessageEvent(const Event_UserMessage &event)
     TabMessage *tab = messageTabs.value(senderName);
     if (!tab)
         tab = messageTabs.value(QString::fromStdString(event.receiver_name()));
-    if (!tab) {
+
+    if (!tab)
+    {
```

Comment:
From our contribution guidelines:

### Braces ###

Use K&R-style braces. Braces for function implementations go on their own
lines, but they go on the same line everywhere else:

    int main()
    {
        if (someCondition) {
            doSomething();
        } else {
            while (someOtherCondition) {
                doSomethingElse();
            }
        }
    }

Braces can be omitted for single-statement if's and the like, as long as it is
still legible.